event-handlers/src/main/java/no/sikt/nva/nvi/events/cristin/CristinMapper.java

```diff
@@ -233,12 +233,16 @@ private static BigDecimal extractBasePoints(CristinNviReport cristinNviReport) {
 
     private List<DbInstitutionPoints> calculatePoints(CristinNviReport cristinNviReport) {
         var institutions = cristinNviReport.cristinLocales();
-        return getCreators(cristinNviReport).stream()
-                   .filter(CristinMapper::hasInstitutionPoints)
-                   .collect(collectToInstitutionOfPoints(institutions))
-                   .stream()
-                   .map(CristinMapper::toDbInstitutionPoints)
-                   .collect(Collectors.toList());
+        if (institutions.isEmpty()) {
+            return List.of();
+        } else {
+            return getCreators(cristinNviReport).stream()
+                       .filter(CristinMapper::hasInstitutionPoints)
+                       .collect(collectToInstitutionOfPoints(institutions))
+                       .stream()
+                       .map(CristinMapper::toDbInstitutionPoints)
+                       .collect(Collectors.toList());
+        }
```

Comment:
When institutions/approvals for nvi candidate are empty, we create candidate without points and approvals.